src/brogue/Items.c

```diff
@@ -2517,6 +2517,20 @@ void itemDetails(char *buf, item *theItem) {
                                     abs(theItem->enchant1 + 1));
                             strcat(buf, buf2);
                             break;
+                        case RING_AWARENESS:
+                            if (theItem->enchant1 > -1) {
+                                sprintf(buf2, "\n\nWith this ring equipped, you will have a %i%% chance to automatically notice adjacent traps and secret doors. (If the ring is enchanted, the chance will %s %i%%.) ",
+                                        min(100, max(0, theItem->enchant1 * 20 + 20)),
+                                        (theItem->enchant1 >= 4 ? "remain" : "increase to"),
+                                        min(100, max(0, theItem->enchant1 * 20 + 40)));
+                                sprintf(buf3, "You may occasionally notice secrets at a distance of up to %i squares away.",
```

Comment:
how occasional does occasionally mean here?

Reply:
either 10% or 20% i think, but i'll have to re-check. and then when a given distance hits 30% chance, you get one more tile of potential search radius, which comes in at 10% chance.

i was thinking that the tooltip might have too many numbers on it at this point, but if this should be specified it can be.

Reply:
I think we at least need to tell the player the search range goes up as well as the strength, yeah if we can quantify that in a useful way, even better